pallets/stateful-storage/src/types.rs

```diff
@@ -1,9 +1,13 @@
 use codec::{Decode, Encode, MaxEncodedLen};
+use common_primitives::{schema::SchemaId, stateful_storage::PageId};
 use frame_support::pallet_prelude::*;
 use scale_info::TypeInfo;
 use sp_core::bounded::BoundedVec;
 use sp_std::{cmp::*, collections::btree_map::BTreeMap, fmt::Debug, prelude::*};
 
+pub type ItemizedKey = (SchemaId,);
```

Comment:
It seems like you have duplicated these types in lib.rs also. I would suggest to define all these types inside here and remove the duplicates defined in `lib.rs`

Reply:
Done.